src/modules/packages/main.py

```diff
@@ -54,7 +54,9 @@ def install(self, pkgs):
             check_chroot_call(["apt-get", "-q", "-y", "install"] + pkgs)
         elif self.backend == "pacman":
             check_chroot_call(["pacman", "-Sy", "--noconfirm"] + pkgs)
-
+# https://wiki.sabayon.org/index.php?title=En:List_Of_Equo_Functions
+                  elif self.backend == "Entropy":
+           check_chroot_call(["equo",  "install"] + pkgs)
```

Comment:
Make sure you are making non-interactive calls to `equo`. Questions like "are you sure you want to install blah blah [Y/n]" from `equo` would block the job.